renv/.gitignore

```diff
@@ -0,0 +1,3 @@
+library/
+python/
```

Comment:
Does the .gitignore have to live in the renv folder, or is this just to make the repo cleaner?

Reply:
Everything in the `renv/` directory is created by the **renv** package (i.e., via `renv::init()` or `renv::snapshot()`). I'm guessing it adds the `.gitignore` to ensure only essential files for reproducing the environment are tracked.